test/services/permissions.test.js

```diff
@@ -9,154 +9,164 @@ describe('services > permissions', () => {
+          });
         });
-      });
-    });
 
-    describe('with some good permissions data on rendering 2', () => {
-      describe('with the "list" permission', () => {
-        it('should return a resolved promise', async () => {
-          expect.assertions(1);
+        describe('check if it requests permissions after a denied access', () => {
+          it('should return a resolved promise', async () => {
+            expect.assertions(1);
 
-          PermissionsChecker.resetExpiration(1);
-          PermissionsChecker.cleanCache();
-          nock.cleanAll();
-          nockObj.get('/liana/v2/permissions?renderingId=2')
-            .reply(200, {
-              Users: {
-                collection: {
-                  list: true,
+            nock.cleanAll();
+            nockObj.get('/liana/v3/permissions?renderingId=1')
+              .reply(200, {
+                meta: { rolesACLActivated: false },
+                data: {
+                  Users: {
+                    collection: {
+                      list: true,
+                    },
+                  },
                 },
-              },
-            });
+              });
 
-          await new PermissionsChecker('envSecret', 2).checkPermissions('Users', 'list')
-            .then(() => { expect(true).toStrictEqual(true); });
+            await new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'browseEnabled', { userId: 1 })
+              .then(() => { expect(true).toStrictEqual(true); });
+          });
         });
       });
-    });
 
-    describe('with some bad permissions data', () => {
-      it('should return a rejected promise', async () => {
-        expect.assertions(1);
+      describe('with some good permissions data on rendering 2', () => {
+        describe('with the "list" permission', () => {
+          it('should return a resolved promise', async () => {
+            expect.assertions(1);
 
-        PermissionsChecker.resetExpiration(1);
-        PermissionsChecker.cleanCache();
-        nock.cleanAll();
-        nockObj.get('/liana/v2/permissions?renderingId=1').reply(200, {});
+            PermissionsChecker.resetExpiration(1);
+            PermissionsChecker.cleanCache();
+            nock.cleanAll();
+            nockObj.get('/liana/v3/permissions?renderingId=2')
+              .reply(200, {
+                meta: { rolesACLActivated: false },
+                data: {
+                  Users: {
+                    collection: {
+                      list: true,
+                    },
+                  },
+                },
+              });
 
-        await expect(new PermissionsChecker('envSecret', 1).checkPermissions('Users', 'list'))
-          .rejects.toThrow("'list' access forbidden on Users");
+            await new PermissionsChecker('envSecret', 2).checkPermissions('Users', 'browseEnabled', { userId: 1 })
+              .then(() => { expect(true).toStrictEqual(true); });
```

Comment:
Looking at your test you should use `.not.toThrow()` which is way more explicit than `expect(true).toStrictEqual(true)` 😅